flink-connector/src/test/java/org/apache/gravitino/flink/connector/integration/test/FlinkCommonIT.java

```diff
@@ -330,4 +331,214 @@ public void testGetSimpleTable() {
         },
         true);
   }
+
+  @Test
+  public void testRenameColumn() {
```

Comment:
seems not include change table comment test?

Reply:
add it.